lib/bot.js

```diff
@@ -36,6 +36,8 @@ class Bot {
     this.channels = _.values(options.channelMapping);
     this.ircStatusNotices = options.ircStatusNotices;
     this.announceSelfJoin = options.announceSelfJoin;
+    this.ignoreBots = options.ignoreBots;
+    this.ignoreIrcNicks = options.ignoreIrcNicks || [];
```

Comment:
Well I needed it for one way. Thought about adding it to Discord->Irc aswell but discord has many ways of identifying a user, I just couldn't choose which one to use.